Add Audinate Dante Via

Trouffman opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?
---


 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/149505)

Trouffman committed:
Add Audinate DanteVia

Trouffman committed:
Uniformize field value and URL between all Dante stuff

stephengillie commented:
Automatic Validation ended with:

> Installation failed with exit code -1978334967
> 2024-04-16 14:35:45.633 [CLI ] ShellExecute installer failed: 3010
> MSI (s) (C0:CC) [14:35:15:809]: Note: 1: 2205 2:  3: Error 
> MSI (s) (C0:CC) [14:35:16:183]: Note: 1: 2205 2:  3: Error 
> MSI (s) (C0:CC) [14:35:16:183]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 1302 
> MSI (s) (C0:CC) [14:35:16:246]: Note: 1: 2205 2:  3: Error 
> MSI (s) (C0:CC) [14:35:16:246]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 1302 
> MSI (s) (C0:CC) [14:35:16:465]: Executing op: ServiceInstall(Name=DanteDiscovery,DisplayName=Dante Discovery,ImagePath="C:\Program Files (x86)\Audinate\Shared Files\mDNSResponder.exe",ServiceType=16,StartType=2,ErrorControl=1,,Dependencies=TCPIP[~][~][~],,,Password=**********,Description=Enables Discovery of Dante Devices,,)
> MSI (s) (C0:CC) [14:35:20:355]: Note: 1: 2205 2:  3: Error 
> MSI (s) (C0:CC) [14:35:20:355]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 22 
> MSI (s) (C0:CC) [14:35:20:355]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 23 
> MSI (s) (C0:CC) [14:35:20:355]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 16 
> MSI (s) (C0:CC) [14:35:20:355]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 21 
> MSI (s) (C0:CC) [14:35:20:527]: Note: 1: 2205 2:  3: Error 
> MSI (s) (C0:CC) [14:35:20:527]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 1707 
> MSI (s) (C0:CC) [14:35:20:527]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 1709 
> MSI (s) (C0:10) [14:35:21:044]: Note: 1: 2205 2:  3: Error 
> MSI (s) (C0:10) [14:35:21:306]: Note: 1: 2205 2:  3: Error 
> MSI (s) (C0:10) [14:35:21:306]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 1302 
> MSI (s) (C0:10) [14:35:21:350]: Note: 1: 2205 2:  3: Error 
> MSI (s) (C0:10) [14:35:21:350]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 1302 
> MSI (s) (C0:10) [14:35:21:551]: Executing op: ServiceInstall(Name=conmon,DisplayName=Dante Control and Monitoring,ImagePath="C:\Program Files (x86)\Audinate\Shared Files\conmon_cmm_service.exe",ServiceType=16,StartType=2,ErrorControl=1,,Dependencies=DanteDiscovery[~][~][~],,,Password=**********,Description=Enables Control and Monitoring of Dante Devices,,)
> MSI (s) (C0:10) [14:35:23:668]: Note: 1: 2205 2:  3: Error 
> MSI (s) (C0:10) [14:35:23:668]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 22 
> MSI (s) (C0:10) [14:35:23:668]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 23 
> MSI (s) (C0:10) [14:35:23:668]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 16 
> MSI (s) (C0:10) [14:35:23:668]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 21 
> MSI (s) (C0:10) [14:35:23:775]: Note: 1: 2205 2:  3: Error 
> MSI (s) (C0:10) [14:35:23:775]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 1707 
> MSI (s) (C0:10) [14:35:23:775]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 1709 
> MSI (s) (C0:F0) [14:35:25:967]: Note: 1: 2205 2:  3: Error 
> C:\lb\win_via_1.3.2.1_release\Dante\rialto_win\installer\src\rialto_wix.cpp.CopyFolder: error opening source folder 'C:\ProgramData\Audinate\Dante Via\': object or item not found: (0x000b=NOTFOUND)
> MSI (s) (C0:F0) [14:35:34:302]: Note: 1: 2205 2:  3: Error 
> MSI (s) (C0:F0) [14:35:34:302]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 22 
> MSI (s) (C0:F0) [14:35:34:302]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 23 
> MSI (s) (C0:F0) [14:35:34:318]: Note: 1: 2205 2:  3: Error 
> MSI (s) (C0:F0) [14:35:34:318]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 16 
> MSI (s) (C0:F0) [14:35:34:318]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 21 
> MSI (s) (C0:F0) [14:35:34:349]: Note: 1: 2205 2:  3: Error 
> MSI (s) (C0:F0) [14:35:34:349]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 1302 
> MSI (s) (C0:F0) [14:35:34:552]: Note: 1: 2205 2:  3: Error 
> MSI (s) (C0:F0) [14:35:34:552]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 1302 
> MSI (s) (C0:F0) [14:35:36:896]: Executing op: ServiceInstall(Name=dantevia.manager,DisplayName=Dante Via Manager,ImagePath="C:\Program Files (x86)\Audinate\Dante Via\DanteViaService.exe",ServiceType=16,StartType=2,ErrorControl=1,,Dependencies=[~],,,Password=**********,Description=Manages Dante Via Components,,)
> C:\lb\win_via_1.3.2.1_release\Dante\rialto_win\installer\src\rialto_wix.cpp.CopyFolder: error opening source folder 'C:\Users\VALIDA~1\AppData\Local\Temp\Rialto Temp\': object or item not found: (0x000b=NOTFOUND)
> Property(S): ErrorDialog = ErrorDlg
> MSI (s) (C0:F0) [14:35:44:927]: Note: 1: 2205 2:  3: Error 
> MSI (s) (C0:F0) [14:35:44:927]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 1707 
> MSI (s) (C0:F0) [14:35:44:927]: Note: 1: 2228 2:  3: Error 4: SELECT `Message` FROM `Error` WHERE `Error` = 1709 

(Automated response - build 876.)

Trouffman commented:
I have been trying to figure this error without success. Even reviewing each log in the pipeline.

stephengillie commented:
Same situation as https://github.com/microsoft/winget-pkgs/pull/149493#issuecomment-2065061165

Trouffman committed:
Update Audinate.DanteVia.installer.yaml

Add support for norestart & 3010 error code to be considered a success.
3010 = reboot required.

Trouffman commented:
Similar fix to consider Error 3010 and adding the /norestart that seems to solve the problem for a similar package : #149493 has been added to this package - same publisher / similar issue.

Trouffman commented:
About the link not matching the publisher domain :
This is the official link as available on their website as : https://my.audinate.com/support/downloads/dante-via

stephengillie commented on manifests/a/Audinate/DanteVia/1.3.2.1/Audinate.DanteVia.locale.en-US.yaml:
```diff
@@ -0,0 +1,22 @@
+PublisherSupportUrl: https://www.getdante.com/support/
+PrivacyUrl: https://www.getdante.com/legal/privacy-policy/
+PackageName: Dante Via
+PackageUrl: https://www.getdante.com/products/software-essentials/dante-via/
```
```suggestion
PackageUrl: https://my.audinate.com/support/downloads/dante-via
```

Trouffman committed:
Update manifests/a/Audinate/DanteVia/1.3.2.1/Audinate.DanteVia.locale.en-US.yaml

Co-authored-by: Stephen Gillie <114952565+stephengillie@users.noreply.github.com>

stephengillie commented:
Verify: 
![image](https://github.com/microsoft/winget-pkgs/assets/114952565/05cf1b41-bbc2-4072-a1b9-321a620d2255)

This installer appears to add multiple applications to the system.
